verification/src/transaction_verifier.rs

```diff
@@ -16,29 +16,38 @@ use ckb_types::{
     prelude::*,
 };
 use std::collections::HashSet;
+use std::convert::TryInto;
 
 /// The time-related TX verification
 ///
 /// Contains:
+/// [`CompatibleVerifier`](./struct.CompatibleVerifier.html)
 /// [`MaturityVerifier`](./struct.MaturityVerifier.html)
 /// [`SinceVerifier`](./struct.SinceVerifier.html)
 pub struct TimeRelativeTransactionVerifier<'a, M> {
+    pub(crate) compatible: CompatibleVerifier<'a>,
```

Comment:
It's a bit weird to put compatible verifier in `TimeRelativeTransactionVerifier`, although you can interpret that time causes incompatibilities.

Reply:
Maybe `CompatibleCountdownVerifier` is a better name?

Reply:
I prefer remove it from `TimeRelativeTransactionVerifier`, unless we believe it is deeply coupled with the since and cell base immature verifier.

Reply:
👌🏼 , and how about first commit? Can I keep it?

I will just refactor the second commit and force push current PR.

Reply:
lgtm